archetypes/example-kar-plugin/src/main/resources/archetype-resources/plugin/src/main/java/AlarmForwarder.java

```diff
@@ -0,0 +1,147 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
```

Comment:
Does the OpenNMS License apply for generated plug ins? i assume when we write yes, for external users - I don't know?

Reply:
Good point - We need to investigate this topic further - Will follow up in OIA-30